common/remote/sockfile.cpp

```diff
@@ -2598,8 +2598,7 @@ void CRemoteFile::copyTo(IFile *dest, size32_t buffersize, ICopyFileProgress *pr
 
 unsigned getRemoteVersion(ISocket * socket, StringBuffer &ver)
 {
-    static CriticalSection sect;
-    CriticalBlock block(sect);
+    // used to have a global critical section here
```

Comment:
I noticed that also and wondered if removing it should be a part of this issue ?  If you think so then I will remove it here as well.